app/views/forms/fields/_person_fields.html.erb

```diff
@@ -1,3 +1,5 @@
+ <% income_fields = 'forms/fields/income_fields' %>
```

Comment:
I will finish the rest tonight.
On May 21, 2016 6:41 PM, "Scott Jones" <notifications@github.com> wrote:

> In app/views/forms/fields/_person_fields.html.erb
> <https://github.com/on-site/grantzilla/pull/265#discussion_r64142776>:
>
> > @@ -1,3 +1,5 @@
> > + <% income_fields = 'forms/fields/income_fields' %>
>
> Good catch; No, it's not...
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/on-site/grantzilla/pull/265/files/de90b99910bdc8613d88bc3b694eee0556647b43#r64142776>
>